src/EditorFeatures/Core/GoToDefinition/GoToDefinitionHelpers.cs

```diff
@@ -58,7 +58,7 @@ internal static class GoToDefinitionHelpers
                 symbol = method.PartialImplementationPart ?? symbol;
             }
 
-            var definitions = ArrayBuilder<DefinitionItem>.GetInstance();
+            using var definitionsDisposer = ArrayBuilder<DefinitionItem>.GetInstance(out var definitions);
```

Comment:
>definitionsDisposer [](start = 22, length = 19)

Don't we want to use `_` here since the variable is not explicitly referenced anywhere?

Reply:
I thought about discarding them but it _feels wrong_. You don't want to discard the disposer, you want to be explicitly aware of it's existence. 

That's just a personal opinion, however.